velox/functions/prestosql/window/tests/AggregateWindowTest.cpp

```diff
@@ -198,5 +198,124 @@ TEST_F(AggregateWindowTest, testDecimal) {
   testAggregate(DECIMAL(5, 2));
   testAggregate(DECIMAL(20, 5));
 }
+
+TEST_F(AggregateWindowTest, integerOverflowRowsFrame) {
+  auto c0 = makeFlatVector<int64_t>({-1, -1, -1, -1, -1, -1, 2, 2, 2, 2});
+  auto c1 = makeFlatVector<double>({-1, -2, -3, -4, -5, -6, -7, -8, -9, -10});
+  // INT32_MAX: 2147483647
+  auto c2 = makeFlatVector<int32_t>(
+      {1,
+       2147483647,
+       2147483646,
+       2147483645,
+       1,
+       10,
+       1,
+       2147483647,
+       2147483646,
+       2147483645});
+  auto c3 = makeFlatVector<int64_t>(
+      {2147483651,
+       2147483650,
+       2147483649,
```

Comment:
Maybe add one more value < INT32_MAX somewhere in between as well.

Reply:
@aditi-pandit, thanks for your suggestion! Just updated.